x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/utils.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiDataGridColumn } from '@elastic/eui';
+
+export const getVulnerabilitiesColumns = (): EuiDataGridColumn[] => {
+  return [
+    {
+      id: 'actions',
+      initialWidth: 40,
```

Comment:
`initialWidth`, `id`, `displayAsText`, can be stored as constants or enums to avoid magic numbers/strings here.